### PR TITLE
Fix for Dockerfile issues building on CentOs 7. See issue #1081

### LINKED ISSUE #1081: Docker not working on Centos 7.
**Expected behavior:**
Build from DockerFile on Centos 7

**Actual behavior:**
Failure with following stacktrace 

> - node-uuid@1.4.7 node_modules/npm/node_modules/request/node_modules/node-uuid
npm ERR! Linux 3.10.0-514.16.1.el7.x86_64
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install" "-g" "npm"
npm ERR! node v6.10.3
npm ERR! npm  v3.10.10
npm ERR! path /usr/lib/node_modules/npm/.github
npm ERR! code ENOTEMPTY
npm ERR! errno -39
npm ERR! syscall rmdir
npm ERR! ENOTEMPTY: directory not empty, rmdir '/usr/lib/node_modules/npm/.github'
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>
npm ERR! Please include the following file with any support request:
npm ERR!     /usr/src/app/npm-debug.log
The command '/bin/sh -c npm install -g npm && npm install -g ember-cli@latest && npm install -g bower' returned a non-zero code: 217

**Steps to reproduce:**
Grab latest from master branch, then try to do `docker build -t hospitalrun-frontend .`

**Screenshots (if applicable):**

**OS and Browser:**
Centos 7

### PR DESCRIPTION
**Pull request by RonanOD**
Fixes #1081 

**Changes proposed in this pull request:**
- Get Dockerfile to build on CentOS 7
- Remove unnecessary npm commands that weren't working

Hi, I solved my difficulties with building the HospitalRun docker image by making these slight changes to the Dockerfile. Hopefully they are acceptable, if not can you let me know what else I should do to get this working. 
Thanks

@tangollama
cc @HospitalRun/core-maintainers

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,8 +9,9 @@ RUN apt-get update && apt-get install nodejs -y
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 COPY . /usr/src/app
-RUN npm install -g npm && npm install -g ember-cli@latest && npm install -g bower
-RUN npm install
+RUN npm install -g ember-cli
```
Comment (jkleinsc) [inline Dockerfile]: I think the only fix needed here is to remove the npm install -g npm.

File: Dockerfile [hunk 2]
```diff
@@ -9,8 +9,9 @@ RUN apt-get update && apt-get install nodejs -y
...
+RUN latest && npm install -g bower
```
Comment (jkleinsc) [inline Dockerfile]: The `latest` should have been on the previous line (eg  npm install -g ember-cli@latest)

File: Dockerfile [hunk 3]
```diff
@@ -9,8 +9,9 @@ RUN apt-get update && apt-get install nodejs -y
...
+#RUN npm install
```
Comment (jkleinsc) [inline Dockerfile]: This line is needed to run HospitalRun

**Comment by RonanOD** [comment]
Thanks @jkleinsc. Made the improvements you suggested and pushed to this branch.

**Pull request merged**